deployments/terraform/modules/sqs/main.tf

```diff
@@ -19,11 +19,13 @@ module "main" {
   visibilityTimeout       = var.visibilityTimeout
   messageRetentionSeconds = var.messageRetentionSeconds
 
-  alarm_create              = var.alarm_main_create
-  alarm_period              = var.alarm_main_period
-  alarm_threshold           = var.alarm_main_threshold
-  alarm_evaluation_periods  = var.alarm_main_evaluation_periods
-  alarm_datapoints_to_alarm = var.alarm_main_datapoints_to_alarm
+  alarm_create                    = var.alarm_main_create
+  alarm_period                    = var.alarm_main_period
+  alarm_visible_message_threshold = var.alarm_main_threshold
+  alarm_evaluation_periods        = var.alarm_main_evaluation_periods
+  alarm_datapoints_to_alarm       = var.alarm_main_datapoints_to_alarm
+  alarm_message_age_create        = var.alarm_main_message_age_create
+  alarm_message_age_threshold     = var.alarm_message_age_threshold
```

Comment:
would still use `var.alarm_main_message_age_threshold` to make it clear that the variable is for the main queue